File: Plov/src/Meat.java

public class Meat {
	
	private int ready = 0;
	
	
	
	private boolean needCut=true;
	private boolean inPan;
	public boolean getInPan() {
		return inPan;
	}
	
	public void setInPan(boolean s) {
		inPan=s;
	}
	
	
	public void setNeedCut(boolean s) {
		needCut=s;
	}
	
	public boolean getNeedCut() {
		return needCut;
	}
	
	public int getReady() {
		return ready;
	}

	public void heat()
	{
		while (ready < 10) ready++;
	}
}

File: Plov/Plov/Meat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Plov
{
    class Meat
    {
        private int ready=0;

        public bool needCut { set; get; }

        public int hasReady
        {

            get
            {
                return ready;
            }
        }

        public void heat()
        {
            while (ready < 10) ready++;
        }
    }
}
